index.js

```diff
@@ -247,6 +247,23 @@ class DHT extends EventEmitter {
     return this._requestToPromise(req, opts)
   }
 
+  delayedPing({ host, port }, delayMs, opts) {
+    const value = b4a.allocUnsafe(4)
```

Comment:
We can throw client-side too for `delayMs > MAX_DELAY`, saves us the round trip for non-malicious clients